mobile/lib/modules/activities/providers/activity.provider.dart

```diff
@@ -95,7 +95,11 @@ class ActivityStatisticsNotifier extends StateNotifier<int> {
   }
 
   Future<void> fetchStatistics() async {
-    state = await _activityService.getStatistics(albumId, assetId: assetId);
+    final count =
```

Comment:
Why make the request at all if not mounted?

Reply:
We'll send the request right after a user opens an album that is shared with other users. However, if the user immediately navigates back to the library page, the response for the request initiated is assigned to the state. However, the widget would've been unmounted and hence, the provider as well, resulting in an error.